OpenIPSL/Examples/Tutorial/Advanced/AutomatingAnalysis.mo

```diff
@@ -0,0 +1,140 @@
+within OpenIPSL.Examples.Tutorial.Advanced;
+model AutomatingAnalysis
+  "Automating Analysis with Modelica Functions"
+  extends Modelica.Icons.Information;
+  annotation (DocumentationClass=true, Documentation(info="<html>
+  <h5>Automating Analysis with Modelica functions</h5>
+  <ol type=\"1\" start=\"1\" style=\"line-height:175%;\">
+    <li>Open the diagram layer of the <font color=\"blue\"><code>SMIB_Partial</code></font> model. Remember that this model is located in the following path: <font color=\"blue\"><code>SMIB.BaseModels.BaseNetwork</code></font>. </li>
+    <li>Double-click the load component and go to the Attributes tab. In the <em>Properties</em> group select the <em>Replaceable</em> option. Then click the Constraining clause checkbox and browse for class by pressing the button <img src=\"modelica://OpenIPSL/Resources/images/example_4/image10.png\" alt=\"Image10\" />. From the package tree select <font color=\"blue\"><code>OpenIPSL.Electrical.Loads.PSSE.BaseClasses.baseLoad</code></font> as shown in the figure below. </li>
+  </ol>
+  <p style=\"margin-left: 50px;\">
+    <img src=\"modelica://OpenIPSL/Resources/images/example_4/image40.png\" alt=\"Image40\" />
+  </p>
+  <ol type=\"1\" start=\"3\">
```

Comment:
The `start` argument is not recognised. Hence the numbering will not work. But why use it in the first place, There is no need for ending and starting several `<ol>` as far as I can see. Simply put the different sections into the respective `<li>`  environments and the display will be correct plus the code more compact.

Reply:
Actually, my bad, The `start` argument does not work in Dymola2020 but works fine in Dymola 2023 and OMEdit 1.20.0. So ignore that. Still I think it is unnecessary and could be cleaner be implemented in one `<ol>` rather than splitting it up into several ones.

Reply:
Alright! I will get rid of the multiple `<ol>` elements.

Reply:
Done!